app.js

```diff
@@ -0,0 +1,40 @@
+//////// Vanilla JS ////////
+// listens for a click
+document.addEventListener('click', function (event) {
```

Comment:
Why delegate the click detection to the entire document?

Reply:
Looking back at it, what I should have done was just focused the click on the form targeting the form id and not the entire page.